content/04.study.md

```diff
@@ -47,6 +47,24 @@ Deep learning applied to gene expression data is still in its infancy, but the f
 Many previously untestable hypotheses can now be interrogated as deep learning enables analysis of increasing amounts of data generated by new technologies.
 For example, the effects of cellular heterogeneity on basic biology and disease etiology can now be explored by single-cell RNA-seq and high-throughput fluorescence-based imaging, techniques we discuss below that will benefit immensely from deep learning approaches.
 
+### DNA Methylation
+
+#### Inference, Imputation, and Prediction
+
+Deep learning approaches are beginning to help address some of the current limitations of feature-by-feature analysis approaches to DNA methylation data, and may help uncover additional important features necessary to understand the biological underpinnings behind different pathological states.
+One of the more popular applications is the prediction of the degree of methylation at CpG sites neighboring measured sites.
+DeepSignal employs a convolutional neural network to construct features from raw electrical Nanopore signals from sites near a methylated base, and concatenates uses a bi-directional recurrent neural network on DNA sequences of the aligned signals to detect methylation [@tag:Ni2018].
+DeepCpG applies a similar method using scBS-Seq, DNA sequence and Bidirectional GRUs [@tag:Angermueller2017] and methods like DAPL and DeepMethyl incorporate sequence and topological structure [@tag:Qiu2018] [@tag:Khwaja2017] [@tag:Wang2016_methyl] [@tag:Fu2019].
+In addition to this, Gene expression has been used to infer and impute methylation states [@tag:Peng2019] [@tag:Levy-Jurgenson2018], methylation of genes predicted from promoter methylation [@tag:Pan2018], and convolutional models have been able to predict methylation status from images [@tag:Momeni2018][@tag:Korfiatis2017].
+While these examples of methylation imputation and inference methods have value it is imperative to recognize limitations of imputing cytosine modifications.
```

Comment:
What is the current state of the art performance for imputation, and is it sufficient for downstream analyses (in your view) or is getting to "useful for many downstream analyses" still a work in progress?

Reply:
I normally just use MICE, K-NN and even Mean imputation, and personally have not tried deep learning imputation approaches, though I am open to developing and implementing new methodologies. I think many of these methods are more geared towards BS-Seq, which can make it harder to adopt for users of 450K and EPIC arrays. Though its conceivable that some of these methods could speed up the analysis, incorporating other modalities may make them more accurate, but coming across this data could still be a challenge. I think making them useful, easy-to-use, and tractable may still be a challenge, but standardized and modular workflows that incorporate these methods may make them more easily adoptable and mainstream.

Reply:
Can you add maybe one or two sentences at the very end of this paragraph around how these methods compare to what's used in practice and whether or not they are at the stage yet where they can replace current methods? From my read of what you wrote, the answer is no because there are still some bespoke processes to get them working on new data (which is not true of other methods). However, you can see a path to get there. Is that right?